packages/samples/headless-ssr-commerce/app/_components/pages/product-page.tsx

```diff
@@ -1,23 +1,24 @@
 'use client';
```

Comment:
I have just seen that there is also an old `product-page.tsx` in `_components/`. I forgot to delete it I think. Do you mind doing it in this PR ?